root/static/scripts/common/entity2.js

```diff
@@ -0,0 +1,364 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+// This file can be renamed to entity.js once that file is removed.
+
+type CommonPropsT = {
+  +id?: number,
+  +name?: string,
+};
+
+export function createNonUrlCoreEntityObject(
```

Comment:
Can this be auto-generated somehow? If not (and maybe even if it can) should we add a comment somewhere to indicate to update this when updating the types?